packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -939,12 +908,13 @@ addFieldsDict(Users, {
     canRead: ['guests'],
     canCreate: ['members'],
     canUpdate: [userOwns, 'sunshineRegiment', 'admins'],
-    group: formGroups.default,
-    label: "Public location (used for your public profile)",
+    group: formGroups.aboutMe,
+    order: 2,
+    label: "Public map location",
     control: 'LocationFormComponent',
     blackbox: true,
     optional: true,
-    order: 43,
+    hidden: true
```

Comment:
In order to create a second form for this collection, I ended up setting all the fields for that form to be `hidden` and then explicitly adding the ones I want to show in my new form, and setting `excludeHiddenFields` to be `false`.